CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix the propagation of replicaCount, installCrds, and pullPolicy values
   in values.yaml during Helm chart installation.
 
-## [24.5.1] - Unreleased
+## [24.5.1] - 2025-01-10
```

Comment:
This change should come from a `release-24.5` forward port, rather than manually doing it